models/marts/structure/fct_staging_directories.sql

```diff
@@ -23,8 +23,8 @@ staging_models as (
         child,
         child_file_path,
         parent_source_name,
-        left(child_file_path, len(child_file_path) - charindex('/',reverse(child_file_path)) + 1) as child_directory_path,
-        {{ dbt_utils.split_part('child_file_path', "'/'", -1) }} as child_file_name
+        replace(child_file_path, child || '.sql', '') as child_directory_path,
```

Comment:
small suggestion to use the [dbt_util for replace](https://github.com/dbt-labs/dbt-utils/blob/main/macros/cross_db_utils/replace.sql) -- may just help future proof this a bit?

Reply:
Changed 😄